packages/bierzo-wallet/src/components/BlockchainAddress/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { Address } from "@iov/bcp";
+import { makeStyles } from "@material-ui/core";
+import { Badge, Typography } from "medulas-react-components";
+import React from "react";
+
+const useStyles = makeStyles({
+  sectionName: {
+    overflowWrap: "break-word",
+  },
+});
+
+interface Props {
+  readonly addresses: Address[];
```

Comment:
`addresses` -> `userAddresses` to make this easier to follow

Type `Address[];` -> `readonly Address[];`

Reply:
What is the difference between `readonly addresses: Address[];` and `readonly addresses: readonly Address[];`?

Reply:
In interfaces, `readonly` means you cannot reassign the value. However, a single array is still mutable, i.e. you can do stuff like `.push` or `.unshift`, causing the array reference to change in all places. This can have unintended effects. `readonly Address[]` is the new native syntax for `ReadonlyArray<Address>`, which is like `Array<Address>` but does not have methods that can change the elements.